artiq/coredevice/phaser.py

```diff
@@ -468,6 +474,32 @@ def get_crc_err(self):
         """
         return self.read8(PHASER_ADDR_CRC_ERR)
 
+    @kernel
+    def measure_frame_timestamp(self):
+        """Perform a register read and record the exact frame timing in `self.frame_tstamp`.
```

Comment:
How it's implemented should not go into the docstring.

Reply:
Changed in 075cb26